specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -190,6 +190,11 @@ Provides settings for capturing photos with CameraCaptureUI.
 The settings include aspect ratio, image size, format, resolution, 
 and whether or not cropping is allowed by the user interface (UI).
 
+**Important Note**:  
+- Don't specify both size and aspect ratio in the `PhotoSettings`, doing so will cause an invalid argument exception.  
+- `PhotoSettings` can't have a ratio or size specified with cropping disabled. Attempting to do so will result in an invalid argument exception.
```

Comment:
Related - should this statement also specify that by default AllowCropping is true? Similar for AllowTrimming?

Reply:
I think it’s a fair point to bring up, adding that clarification could help prevent confusion if the developer is unaware of the defaults.